src/ocean/math/Math.d

```diff
@@ -1715,7 +1715,9 @@ real hypot(real x, real y)
     static immutable real SQRTMIN = 0x8.0p-8195L; // 0.5 * sqrt(real.min_normal); // This is a power of 2.
     static immutable real SQRTMAX = 1.0L / SQRTMIN; // 2^^((max_exp)/2) = nextUp(sqrt(real.max))
 
-    static assert(2 * (SQRTMAX / 2) * (SQRTMAX / 2) <= real.max);
+    // TODO: annotation for support of MIPS64
```

Comment:
What does MIPS64 have to do with this ?

Reply:
real.nan largest FP size implemented in hardwareImplementation Note: 80 bits for x86 CPUs or double size, whichever is larger
I saw Dlang's note.
I don't think it's an OS problem. So I don't know exactly what hardware I'm referring to.
So I changed the annotation to flexible hardware.